build.gradle

```diff
@@ -237,6 +243,7 @@ configure(javaProjects) {
         testCompile 'org.apache.httpcomponents:httpclient'
         testCompile 'net.javacrumbs.json-unit:json-unit'
         testCompile 'net.javacrumbs.json-unit:json-unit-fluent'
+        testCompile  "org.dmonix.junit:zookeeper-junit"
```

Comment:
- double space -> single space
- `"` -> `'`
- Please move this dependency to the `build.gradle` of the module that actually uses ZooKeeper JUnit, given that this dependency is not used by other modules.